contracts/toCKB-typescript/src/switch/mod.rs

```diff
@@ -132,6 +136,32 @@ fn get_deletion_tx_type(data: &ToCKBCellDataView) -> Result<TxType, Error> {
     }
 }
 
+fn verify_xt(tx_type: &TxType) -> Result<(), Error> {
+    use TxType::*;
+    match tx_type {
+        MintXT
+        | PretermRedeem
+        | AttermRedeem
+        | AuctionSignerTimeout
+        | AuctionUnderCollateral
+        | AuctionFaultyWhenWarranty
+        | AuctionFaultyWhenRedeeming => Ok(()),
+        _ => forbid_mint_xt(),
+    }
+}
+
+fn forbid_mint_xt() -> Result<(), Error> {
+    let sudt_cell_count = QueryIter::new(load_cell_type, Source::Output)
+        .filter(|type_opt| type_opt.is_some())
+        .map(|type_opt| type_opt.unwrap())
+        .filter(|script| script.code_hash().raw_data().as_ref() == SUDT_CODE_HASH.as_ref())
```

Comment:
```suggestion
        .filter(|script| script.code_hash().raw_data().as_ref() == SUDT_CODE_HASH.as_ref() 
          || script.args().raw_data().as_ref() == toCKB_lock_hash )
        
```
the added code is aim to forbid user mint fake sudt token of which code_hash != SUDT_CODE_HASH and args == toCKB_lock_hash

Reply:
there is no need to consider fake sudt